src/main/java/growthcraft/milk/common/tileentity/TileEntityButterChurn.java

```diff
@@ -217,26 +193,22 @@ public boolean tryTakeItem(IItemOperable.Action action, EntityPlayer player, Ite
     @TileEventHandler(event = TileEventHandler.EventType.NBT_READ)
     public void readFromNBT_ButterChurn(NBTTagCompound nbt) {
         this.shaftState = nbt.getInteger("shaft_state");
-        this.churns = nbt.getInteger("churns");
     }
 
     @TileEventHandler(event = TileEventHandler.EventType.NBT_WRITE)
     public void writeToNBT_ButterChurn(NBTTagCompound nbt) {
         nbt.setInteger("shaft_state", shaftState);
-        nbt.setInteger("churns", churns);
     }
 
     @TileEventHandler(event = TileEventHandler.EventType.NETWORK_READ)
     public boolean readFromStream_ButterChurn(ByteBuf stream) throws IOException {
         this.shaftState = stream.readInt();
-        this.churns = stream.readInt();
```

Comment:
With this change clients will need to be running on the same version or later when this change is merged, just a note

Reply:
I didn't know you can have a different version on client and server. If this is a problem i can leave this line.